src/wallet/wallet.cpp

```diff
@@ -556,19 +555,26 @@ std::set<uint256> CWallet::GetConflicts(const uint256& txid) const
         return result;
     const CWalletTx& wtx = it->second;
 
+    result = GetConflicts(*(wtx.tx), true);
+
+    return result;
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

`result` is not needed here.

```suggestion
    return Getconflicts(*wtx.tx, true);
````